src/govuk/components/select/_index.scss

```diff
@@ -7,6 +7,7 @@
     @include govuk-font($size: 19, $line-height: 1.25);
 
     box-sizing: border-box; // should this be global?
+    min-width: 23ex;
```

Comment:
I think your reasoning in the PR is sound. Is it worth trying to summarise that in a comment here?

Reply:
Added a code comment parroting my comment above.

Also added a changelog entry, currently listed as a fix, though I suppose this could also be a new feature, in a sense??

Reply:
Great, cheers! I see what you mean about it being a new feature potentially but my instinct is to keep it a fix as it's technically fixing the problem of the select not having a min-width.